homeassistant/components/evohome/__init__.py

```diff
@@ -482,6 +486,8 @@ def get_session_id(client_v1) -> Optional[str]:
             async_dispatcher_send(self.hass, DOMAIN)
 
             _LOGGER.debug("Status = %s", status[GWS][0][TCS][0])
+            # TODO: also update utc offset...
```

Comment:
To do

Reply:
Yeah - just waiting for testers before I go further.